fusillade/config.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+
+import boto3
+
+
+class Config:
+    _admin_emails = None
+    _oauth2_config = None
+
+    @classmethod
+    def get_admin_emails(cls):
+        if not cls._admin_emails:
+            cls._admin_emails = [admin for admin in os.environ['FUS_ADMIN_EMAILS'].split(',')]
```

Comment:
`cls._admin_emails = [admin.strip() for admin in os.environ['FUS_ADMIN_EMAILS'].split(',')]`
Nit: Might be worthwhile to use `.strip()` to be safe.  ^

Reply:
https://github.com/HumanCellAtlas/data-store/blob/master/scripts/populate_lambda_ssm_parameters.py#L81 might be of use

Reply:
will fix